Dockerfile

```diff
@@ -26,7 +26,18 @@ RUN mvn clean package -DskipTests=true -Dspotbugs.skip=true -Dpmd.skip=true
 
 # Runtime Container
 FROM amazoncorretto:17-alpine
+
+# Setup rights for overwriting frontend runtime variables
+COPY --from=VUE /frontend/setup-runtime.sh setup-runtime.sh
+
+RUN \
+    mkdir public && \
```

Comment:
Why the public folder, are the contents of the .jar extracted there?

Reply:
The public folder is necessary because the jar tool ("jar uf") requires the same path of the file to modify the file in the existing jar file. See: https://docs.oracle.com/javase/tutorial/deployment/jar/update.html
There does not seem to be way around this (the jar uf -C option does not provide an alternative)